SPECS/azurelinux-rpm-macros/azurelinux-rpm-macros.spec

```diff
@@ -43,6 +42,7 @@ Source24:       macros.suse
 Source25:       gen-ld-script.sh
 Source26:       generate-package-note.py
 Source27:       verify-package-notes.sh
+Source28:       https://src.fedoraproject.org/rpms/python-rpm-macros/blob/f40/f/python.lua
```

Comment:
Fedora includes that file in its `python-srpm-macros` package (subpackage of `python-rpm-macros.spec`). I know we agreed to adding that package ourselves and moving out the files provided by it from this spec, but could we group all files in this spec, which would eventually come from `python-rpm-macros` (`macros.python`, `macros.pybytecompile`, `macros.python-srpm`, etc.) and add a comment saying where we can look for newer version, which would be compatible with each other?

Reply:
I updated the PR to group these files together, add source URLs, and update to the latest contents from Fedora 40